lib/user_agent/browsers/wondery.rb

```diff
@@ -0,0 +1,91 @@
+class UserAgent
+  module Browsers
+    # wondery/android/v1.8.2/2314
+    # Wondery/1660 CFNetwork/1183 Darwin/20.0.0
+    # wondery/ios/v1.8.2/48
+    # wondery/ios/v1.4.2/1 ,AppleCoreMedia/1.0.0.17G64 (iPhone; U; CPU OS 13_6 like Mac OS X; en_us)
```

Comment:
wondery sometimes uses UAs with staging or development added. Maybe you want to reach out to them in our separate slack channel to confirm this is good enough.